diff --git a/node_modules/folks-finance-js-sdk/src/v1/lend/constants.ts b/node_modules/folks-finance-js-sdk/src/v1/lend/constants.ts
index 9cc17e4..08a23fe 100644
--- a/node_modules/folks-finance-js-sdk/src/v1/lend/constants.ts
+++ b/node_modules/folks-finance-js-sdk/src/v1/lend/constants.ts
@@ -240,7 +240,7 @@ const TestnetOracle: Oracle = {
 
 const TestnetReserveAddress = "5ISPVI3JMQ4MP5XWWK4ILLKJAYU34U4TGZGXNC5BKR5AK3JBVAMJYBRYEI";
 
-export {
+export type {
   TestnetPoolsKey,
   TestnetPools,
   TestnetTokenPairsKey,
